### DISCUSSION TITLE
Nanite and Lumen in UE5 are amazing. Will o3de have something similar in the near future?

### DISCUSSION THREAD

**Discussion by ThisGitHubUsernameWasAvailable** [o3de/o3de#6737] [General]
Nanite and Lumen in UE5 are amazing. Will o3de have something similar in the near future?

**Comment by SacredTotems** [comment]
I linked this thread in the Open 3D discord & a mod replied with this;   " As a side note, the engine technically already has Lumen. That Lumen replaces the light baking that UE4 had, while O3DE already has real-time rendering. "

**Comment by amzn-tommy** [comment]
O3DE does not have 'Lumen' per-se, since that is a specific solution to global illumination that was developed for UE5. But O3DE has a global illumination solution using DDGI. https://o3de.org/docs/user-guide/components/reference/atom/diffuse-probe-grid/

Nanite has come up in the sig-graphics-audio discord channel as well https://discord.com/channels/805939474655346758/816043793576886273/918517363752702013

Re-posting my opinion here: 
In addition to the hierarchical lod setup that makes Nanite possible, Nanite is a culmination of a number of other geometry pipeline technologies that are standard in a lot of AAA graphics pipelines, like mesh optimization, streaming, compression, meshlets, GPU culling, bindless, etc. The way I think about it is that each of these brings their own benefits individually, and we need to start incrementally and iteratively getting these parts right in the near term, which will bring immediate benefits to O3DE, while having a long term plan for how it all fits together for a state of the art geometry pipeline.

Ultimately, it's up to sig-graphics-audio to discuss and decide how to chart the course.
